test/acceptance/Readme.md

```diff
@@ -0,0 +1,59 @@
+# Orion Acceptance Tests
+
+Folder for acceptance tests of Orion.
+
+## How to Run the Acceptance Tests
+
+### Prerequisites:
+
+- Python 2.7
+- pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
+- virtualenv installed (pip install virtualenv) (optional).
+Note: We recommend the use of virtualenv, because is an isolated working copy of Python which allows you to work on a specific project without worry of affecting other projects.
+
+##### Environment preparation:
+
+- If you are going to use a virtual environment (optional):
```

Comment:
Ah, ok, I didn't realize about it.

Maybe is better to use the "code style" for things that are actually literal commands, I mean:

```
Create a virtual environment somewhere, e.g. in ~/venv (`virtualenv ~/venv`)
Activate the virtual environment (`source ~/venv/bin/activate`) (optional)
```